inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/codec/ProtocolEncoder.java

```diff
@@ -134,9 +135,9 @@ private ByteBuf constructBody(byte[] body, EncodeObject object,
                     }
                     EncryptInfo encryptInfo = encryptEntry.getRsaEncryptInfo();
                     endAttr = endAttr + "_userName=" + object.getUserName()
-                        + "&_encyVersion=" + encryptInfo.getVersion()
-                        + "&_encyDesKey=" + encryptInfo.getRsaEncryptedKey();
-                    body = EncryptUtil.desEncrypt(body, encryptInfo.getDesKey());
+                            + "&_encyVersion=" + encryptInfo.getVersion()
+                            + "&_encyAesKey=" + encryptInfo.getRsaEncryptedKey();
+                    body = EncryptUtil.aesEncrypt(body, encryptInfo.getDesKey());
```

Comment:
It is not fit that getDesKey is a AES Key.
body = EncryptUtil.aesEncrypt(body, encryptInfo.getDesKey());